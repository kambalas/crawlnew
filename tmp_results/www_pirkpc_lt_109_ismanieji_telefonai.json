[
  {
    "name": "Apple iPhone 16 Pro Max | Desert Titanium | 6.9 \" | Super Retina XDR | 2868 x 1320 pixels | A18 Pro | Internal RAM 8 GB | 512 GB | Dual SIM | 4G | 5G | Main camera resolution 48+12 MP | Secondary camera resolution 12 MP | iOS",
    "price": "1 672,27 €",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro | Desert Titanium | 6.3 \" | Super Retina XDR display with ProMotion | 2622 x 1206 pixels | A18 Pro | Internal RAM 8 GB | 512 GB | Dual SIM | 4G | 5G | Main camera resolution 48+12 MP | Secondary camera resolution 12 MP | iOS | 18",
    "price": "1 551,38 €",
    "error": false
  },
  {
    "name": "Apple iPhone 16 | Black | 6.1 \" | Super Retina XDR | 2556 x 1179 pixels | A18 | Internal RAM 8 GB | 512 GB | Dual SIM | 4G | 5G | Main camera resolution 48+12 MP | Secondary camera resolution 12 MP | iOS | 18",
    "price": "1 285,92 €",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Plus | Ultramarine | 6.7 \" | Super Retina XDR | 2796 x 1290 pixels | A18 | Internal RAM 8 GB | 128 GB | Dual SIM | 4G | 5G | Main camera resolution 48+12 MP | Secondary camera resolution 12 MP | iOS | 18",
    "price": "1 052,30 €",
    "error": false
  },
  {
    "index": 0,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "'str' object has no attribute 'choices'"
  }
]