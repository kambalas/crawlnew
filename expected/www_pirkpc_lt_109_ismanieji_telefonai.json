[
  {
    "name": "iPhone 13 Pro Max 256GB Sierra Blue (MGD63ZE/A)",
    "price": "1289",
    "error": false
  },
  {
    "name": "iPhone 13 Pro Max 128GB Sierra Blue (MGD53ZE/A)",
    "price": "1179",
    "error": false
  },
  {
    "name": "iPhone 13 Pro Max 512GB Sierra Blue (MGD73ZE/A)",
    "price": "1479",
    "error": false
  },
  {
    "name": "iPhone 13 Pro Max 256GB Graphite (MGD84ZE/A)",
    "price": "1289",
    "error": false
  },
  {
    "name": "iPhone 13 Pro Max 128GB Graphite (MGD54ZE/A)",
    "price": "1179",
    "error": false
  },
  {
    "name": "iPhone 13 Pro Max 512GB Graphite (MGD74ZE/A)",
    "price": "1479",
    "error": false
  },
  {
    "index": 1,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "'str' object has no attribute 'choices'"
  }
]